csrank/objectranking/fate_object_ranker.py

```diff
@@ -1,95 +1,85 @@
-            :math:`U (x, \\mu_{C(x)})`.
-            To make it computationally efficient we take the the context :math:`C(x)` as query set :math:`Q`.
-            The context-representation is evaluated as:
-
-            .. math::
-                \\mu_{C(x)} = \\frac{1}{\\lvert C(x) \\lvert} \\sum_{y \\in C(x)} \\phi(y)
-
-            where :math:`\\phi \\colon \\mathcal{X} \\to \\mathcal{Z}` maps each object :math:`y` to an
-            :math:`m`-dimensional embedding space :math:`\\mathcal{Z} \\subseteq \\mathbb{R}^m`.
-            Training complexity is quadratic in the number of objects and prediction complexity is only linear.
-            The ranking for the given query set :math:`Q` is defined as:
-
-            .. math::
-                ρ(Q)  = \\operatorname{argsort}_{x \\in Q}  \\; U (x, \\mu_{C(x)})
-
-            Parameters
-            ----------
-            n_hidden_set_layers : int
-                Number of set layers.
-            n_hidden_set_units : int
-                Number of hidden set units.
-            n_hidden_joint_layers : int
-                Number of joint layers.
-            n_hidden_joint_units : int
-                Number of joint units.
-            activation : string or function
-                Activation function to use in the hidden units
-            kernel_initializer : function or string
-                Initialization function for the weights of each hidden layer
-            kernel_regularizer : uninitialized keras regularizer
-                Regularizer to use in the hidden units
-            optimizer: Class
-                Uninitialized optimizer class following the keras optimizer interface.
-            optimizer__{kwarg}
-                Arguments to be passed to the optimizer on initialization, such as optimizer__lr.
-            batch_size : int
-                Batch size to use for training
-            loss_function : function
-                Differentiable loss function for the score vector
-            metrics : list
-                List of evaluation metrics (can be non-differentiable)
-            random_state : int or object
-                Numpy random state
-            **kwargs
-                Keyword arguments for the @FATENetwork
-        """
-        self.loss_function = loss_function
-        self.metrics = metrics
-        super().__init__(
-            n_hidden_set_layers=n_hidden_set_layers,
-            n_hidden_set_units=n_hidden_set_units,
-            n_hidden_joint_layers=n_hidden_joint_layers,
-            n_hidden_joint_units=n_hidden_joint_units,
-            activation=activation,
-            kernel_initializer=kernel_initializer,
-            kernel_regularizer=kernel_regularizer,
-            optimizer=optimizer,
-            batch_size=batch_size,
-            random_state=random_state,
-            **kwargs,
+        self.n_hidden_set_layers = n_hidden_set_layers
+        self.n_hidden_set_units = n_hidden_joint_units
```

Comment:
`self.n_hidden_set_units = n_hidden_set_units` ;)

Reply:
Fixed, thanks for noticing.